task/01-strings-tasks.js

```diff
@@ -145,7 +158,10 @@ function removeFirstOccurrences(str, value) {
  *   '<a>' => 'a'
  */
 function unbracketTag(str) {
-    throw new Error('Not implemented');
+    const startIndex = 1
+    const endIndex = str.length - 1
+    const newStr = str.slice(startIndex, endIndex)
+    return newStr
```

Comment:
```suggestion
    return str.slice(1, -1)
```

Looks much cleaner and readable, isn't it?